### PR TITLE
League post design

### LINKED ISSUE #121: [OPTIONAL] Angular: Error page

### LINKED ISSUE #144: Add Leage post design
- #7

### LINKED ISSUE #145: Create date ago pipe to transform dates

### LINKED ISSUE #147: Add loading spinner to League post
- #7

### PR DESCRIPTION
**Pull request by dudaskobendeguz**
-  closes #144 
- closes #145 
- closes #147

### CODE REVIEW

File: angular/src/app/components/date-ago/date-ago.component.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<span *ngIf="date" matTooltip="{{ date | date : 'yyyy.MMMM d., EEEE, HH:mm' }}">{{ date | dateAgo }}</span>
```
Comment (benedekhalaj) [inline angular/src/app/components/date-ago/date-ago.component.html:1]: This is pure beauty

File: angular/src/app/components/league-posts/icons/icons.component.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  <div class="players">
+    <mat-icon matTooltip="Players">supervisor_account</mat-icon>
+    <span>{{ numberOfPlayers }}</span>
+  </div>
+
+  <div class="rounds">
+    <mat-icon matTooltip="Rounds">settings_backup_restore</mat-icon>
+    <span>{{ numberOfRounds }}</span>
+  </div>
+</div>
```
Comment (benedekhalaj) [inline angular/src/app/components/league-posts/icons/icons.component.html:16]: Great structure

File: angular/src/app/components/league-posts/league-post-detail/league-post-detail.component.css [hunk 3]
```diff
@@ -1,3 +1,9 @@
+.league-post-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
```
Comment (benedekhalaj) [inline angular/src/app/components/league-posts/league-post-detail/league-post-detail.component.css:4]: simple and great

File: angular/src/app/pipes/date-ago/date-ago.pipe.ts [hunk 4]
```diff
@@ -0,0 +1,39 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'dateAgo'
+})
+export class DateAgoPipe implements PipeTransform {
+
+  private intervals: {[key: string]: number} = {
+    'year': 60*60*24*365,
+    'month': 60*60*24*30,
```
Comment (benedekhalaj) [inline angular/src/app/pipes/date-ago/date-ago.pipe.ts:10]: It is nice to see such a useful pipe in our application :) I'm super glad

File: angular/src/styles.css [hunk 5]
```diff
@@ -1,5 +1,10 @@
 /* You can add global styles to this file, and also import other style files */
 
+:root {
+  --post-width: 630px;
+  --post-gap: 30px;
+}
```
Comment (benedekhalaj) [inline angular/src/styles.css:6]: Also, I like your progress on CSS, great additions :)

**Review by benedekhalaj — APPROVED** [review]
Excellent work!

File: angular/src/app/components/league-posts/league-post-detail/league-post-detail.component.css (hunk 3 above)
Comment (dudaskobendeguz) [reply angular/src/app/components/league-posts/league-post-detail/league-post-detail.component.css:4]: I have to finish it, but thank you :+1:

File: angular/src/app/pipes/date-ago/date-ago.pipe.ts (hunk 4 above)
Comment (dudaskobendeguz) [reply angular/src/app/pipes/date-ago/date-ago.pipe.ts:10]: Thank you :). I'm so proud for that .... Feel free to use it :dancers:

File: angular/src/styles.css (hunk 5 above)
Comment (dudaskobendeguz) [reply angular/src/styles.css:6]: I had a great CSS mentor:  @benedekhalaj :dagger:

**Pull request merged**